en/04_Changelogs/5.0.0.md

```diff
@@ -205,6 +205,9 @@ This is a major release and as a result there are a number of breaking API chang
 their aliases (applied by setting a string key for the array) were being ignored. This bug has been fixed - if you were working around this by manually setting the alias e.g. in a join, you can remove those workarounds now.
 - [Query](api:SilverStripe\ORM\Connect\Query) now implements `IteratorAggregate` instead of `Iterator`. This means `seek()` and other iterator methods are no longer available on this class and its subclasses. Use `getIterator()` instead.
 - [DataList](SilverStripe\ORM\DataList), its subclasses, [Map](api:SilverStripe\ORM\Map), and [ArrayList](SilverStripe\ORM\ArrayList) all now return generators from `getIterator()`. This reduces memory usage when looping over large result sets. As a result of this, `getGenerator()` has been removed as it is no longer needed. Note that `chunkedFetch()` has not been removed, as it may still be useful for very large result sets to fetch results in smaller chunks at a time.
+- [Limitable::limit()](SilverStripe\ORM\Limitable::limit()) is now strongly typed. Calling `limit()` with a `0` or `false` length now limits the list to 0 entry. In Silverstripe CMS 4, any "falsy" value would unset the limit. To unset a list's limit in Silverstripe CMS 5 , call `limit()` with an explicit `null`. This affects [DataList](SilverStripe\ORM\DataList), [ArrayList](SilverStripe\ORM\ArrayList) and all other classes implementing Limitable](SilverStripe\ORM\Limitable)
+  - Calling [SQLSelect::setLimit()](SilverStripe\ORM\Queries\SQLSelect::setLimit()) with `0` used to unset the limit. It will now set a limit of `0` causing the query to return no results. `setLimit()` with `null` to unset the limit.
```

Comment:
```suggestion
  - In Silverstripe CMS 4, calling [SQLSelect::setLimit()](SilverStripe\ORM\Queries\SQLSelect::setLimit()) with `0` as limit argument was used to unset the limit. in Silverstripe CMS 5, set a limit of `0` will be causing the query to return no results. Now, to unset the limit use `setLimit()` with `null` as limit value.
```

Reply:
Could we paraphrase this sentence to make it more accessible for understanding?

Reply:
```suggestion
  - In Silverstripe CMS 4, calling [SQLSelect::setLimit()](SilverStripe\ORM\Queries\SQLSelect::setLimit()) with `0` as argument would unset the limit. In Silverstripe CMS5, it sets a limit of `0` causing the query to return no results. Call `setLimit()` with `null` to unset the limit.
```

Reply:
Tweaked your suggestion a bit, but otherwise kept most of it.